### DISCUSSION TITLE
ivcam green screen mode recording freeze problem

### DISCUSSION THREAD

**Discussion by star2k** [obsproject/obs-studio#6370] [Development Questions] [answered]
When the obs records the ivcam virtual camera video, when the ivcam sets the green screen background, the video recorded in the obs will be obviously stuck, but the software that comes with the ivcam is normal

**Comment by Fenrirthviti** [comment, marked as answer]
This sounds more likely to be a support request, and we are not currently accepting support requests on GitHub Discussions. Make sure you read any [pinned posts](https://github.com/obsproject/obs-studio/discussions/5206) on discussion boards before opening a new discussion. Please use our [forums](https://obsproject.com/forum/) or [Discord](https://obsproject.com/discord/) for further assistance.

Thank you!
